approval_polls/views.py

```diff
@@ -30,12 +33,19 @@ def getPolls(request, poll_list, render_page):
       polls = paginator.page(paginator.num_pages)
   return render(request, render_page, {'latest_poll_list' : polls})
 
-class DetailView(generic.DetailView):
+class DetailView(generic.DeleteView):
   model = Poll
   template_name = 'approval_polls/detail.html'
   def get_queryset(self):
       return Poll.objects.filter(pub_date__lte=timezone.now())
 
+  @staticmethod
```

Comment:
@ashimaathri 
I've read through this and getting clearer idea regarding why we use a class-based view to manage the resource. Now I just have some minor issues to discuss with you:
- It seems like not all browser support DELETE and PUT for ajax request, so I just pass the type as POST to make my code work (passing DELETE does not work on my machine).
- I've tried both DetailView and DeleteView, it looks like delete does not work under DetailView but under DeleteView, that's why I use DeleteView. After digging into the issue I found this:
https://ccbv.co.uk/projects/Django/1.7/django.views.generic.detail/DetailView/
https://ccbv.co.uk/projects/Django/1.7/django.views.generic.edit/DeleteView/

I know it might not be what you expect, so please let me know your thought if I misunderstood anything.
Thanks!